### PR TITLE
History.md - add links [skip ci]

### PR DESCRIPTION
**Pull request by MSP-Greg**
### Description

Add links to History.md.

Leave the title attribute (date & user)?

I've got code for the first pass, I'll get it working for updates (grabs repo pr/issue info with GraphQL call).

### Your checklist for this pull request

- [x] I have reviewed the [guidelines for contributing](../blob/master/CONTRIBUTING.md) to this repository.
- [x] I have added an entry to [History.md](../blob/master/History.md) if this PR fixes a bug or adds a feature. If it doesn't need an entry to HISTORY.md, I have added `[changelog skip]` or `[ci skip]` to the pull request title.
- [ ] I have added appropriate tests if this PR fixes a bug or adds a feature.
- [ ] My pull request is 100 lines added/removed or less so that it can be easily reviewed.
- [ ] If this PR doesn't need tests (docs change), I added `[ci skip]` to the title of the PR.
- [ ] If this closes any issues, I have added "Closes `#issue`" to the PR description or my commit messages.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed, including Rubocop.

### CODE REVIEW

File: History.md [hunk 1]
```diff
@@ -119,107 +121,107 @@ Each patchlevel release contains a separate security fix. We recommend simply up
...
+  * Change pid existence check's condition branches ([#1650])
+  * Don't call .stop on a server that doesn't exist ([#1655])
+  * Implemented NID_X9_62_prime256v1 (P-256) curve over P-521 ([#1671])
+  * Fix @notify.close can't modify frozen IOError (RuntimeError) ([#1583])
+  * Fix Java 8 support ([#1773])
+  * Fix error `uninitialized constant Puma::Cluster` ([#1731])
+  * Fix `not_token` being able to be set to true ([#1803])
+  * Fix "Hang on SIGTERM with ruby 2.6 in clustered mode" ([PR [#1741]], [#1674], [#1720], [#1730], [#1755])
+
+[PR [#1741]]: https://github.com/puma/puma/pull/1741
```
Comment (dentarg) [inline History.md:224]: Not sure what's wrong, but the rendered file doesn't look good here:

Screenshot 2020-09-25 at 07 52 37

**Comment by dentarg** [comment]
I like this 👍

**Comment by MSP-Greg** [comment]
@dentarg

Sorry, I assumed that the first pass thru it contained no links.  Fixed, reformatted some other things, I think it's correct now.

**Review by dentarg — APPROVED** [review]
LGTM 👍 

@MSP-Greg do we expect contributors to make links or will you run your tooling (sounds like you have something) every now and then?

**Comment by MSP-Greg** [comment]
@dentarg

> do we expect contributors to make links

Please, no.  That presupposes a knowledge of markdown, etc...

I'll work on the tooling, maybe add it to tools/history or something.  I may need to add a json file that contains info about every PR and issue, as one never knows how far back they are, especially with issues.  Right now, the file is about 450k.  It also requires that one has a token to access the GH GraphQL API.

Also, the tooling could go in another Puma repo...

**Comment by nateberkopec** [comment]
Beautiful!

**Pull request merged**